test/integration/util/minikube_runner.go

```diff
@@ -229,14 +228,21 @@ func (m *MinikubeRunner) SSH(cmdStr string) (string, error) {
 }
 
 // Start starts the cluster
-func (m *MinikubeRunner) Start(opts ...string) (stdout string, stderr string, err error) {
+func (m *MinikubeRunner) start(opts ...string) (stdout string, stderr string, err error) {
 	cmd := fmt.Sprintf("start %s %s %s", m.StartArgs, m.GlobalArgs, strings.Join(opts, " "))
-	s := func() error {
-		stdout, stderr, err = m.RunCommandRetriable(cmd)
-		return err
+	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(ctx, m.TimeOutStart)
+	defer cancel()
+	return m.RunWithContext(ctx, cmd, true)
+}
+
+// StartWithFail starts the cluster and fail the test if error
+func (m *MinikubeRunner) StartWithFail(opts ...string) (stdout string, stderr string) {
```

Comment:
The standard Go nomenclature for functions that fatally exit is `MustStart`. Usually those panic, but I think it would read well here too. I wasn't sure what `StartWithFail` would do.